### ISSUE TITLE
Make dimmer a Close button in VO mode

### ISSUE THREAD

**Issue by mpospese** [yml-org/ybottomsheet-ios#11]
## Intro
In Voice Over mode we should make the transparent area above the sheet a button to enable it to be explicitly tapped.
We basically already have this ability in non-VO mode (tap anywhere on dimmer to dismiss), we just need to enable it as an explicit button when VO is enabled.

## Task
* Add a child view that will be pinned to top and sides of BottomSheetController.view and have its bottom pinned to the top of `sheetView`. This will give it a frame of the space above the sheet (and will change as user resizes or the content grows).
* **Important:** None of this should apply when `appearance.isDismissAllowed == false`.
* We should probably move the tap gesture recognizer from the dimmer to this view
* It should have the `.button` accessibility trait
* It should have accessibilityLabel = "Dismiss" (same string as the close button)
* It should have no accessibilityHint
* It should have an accessibilityIdentifier = `sheet.dimmer.close`
* (We should add an identifier to our default close button of `sheet.button.close`):
```
/// Accessibility Identifiers
enum AccessibilityIdentifiers {
    /// Button ID
    static let buttonId = "sheet.button.close"
    /// Dimmer ID
    static let dimmerId = "sheet.dimmer.close"
}
```
